services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/config/JdbcProperties.java

```diff
@@ -168,14 +176,40 @@ public static JDBCClient dataSource(final Vertx vertx, final JdbcProperties data
 
         log.info("Creating new SQL client: {} - table: {}", config, dataSourceProperties.getTableName());
 
-        // put password after logging
-
-        config
-            .put("password", dataSourceProperties.getPassword());
-
         // create new client
 
-        return JDBCClient.create(vertx, config);
+        final int minSize = config.getInteger(minSizeLabel);
+        final int maxSize = config.getInteger(maxSizeLabel);
+        final int initSize = config.getInteger(initSizeLabel);
+        final Duration connectionTimeout = Duration.ofSeconds(30);
```

Comment:
I think we shouldn't hard set this value but rather make it configurable like the other values with a reasonable default value.

Reply:
True, I will make this connection timeout configurable also.